ts/adaptors/liteAdaptor.ts

```diff
@@ -568,7 +594,12 @@ export class LiteAdaptor extends AbstractDOMAdaptor<LiteElement, LiteText, LiteD
    */
   public nodeSize(node: LiteElement, _em: number = 1, _local: boolean = null) {
     const text = this.textContent(node);
-    return [.6 * text.length, 0] as [number, number];
+    const non = text.replace(LiteAdaptor.cjkPattern, '').split('').length; // # of non-CJK chars
```

Comment:
Counting with `length` could give the wrong number of characters if there are other characters from an upper Unicode plane in the string.

Reply:
Right.  I thought it was unicode safe, but I was wrong.  I should have used `Array.from()`, which handles unicode character properly.  I will make another commit for it.